code/datums/objective.dm

```diff
@@ -381,6 +381,22 @@ proc/create_fluff(var/datum/mind/target)
 
 /datum/objective/regular/bonsaitree
 	// Brought this back as a very rare gimmick objective (Convair880).
+#ifdef MAP_OVERRIDE_MANTA
+	explanation_text = "Destroy the Captain's ship in a bottle."
+
+	check_completion()
+		var/area/cap_quarters = locate(/area/station/captain)
+		var/obj/captain_bottleship/cap_ship
+
+		for (var/obj/captain_bottleship/T in cap_quarters)
```

Comment:
Probably out of the scope of this PR but it'd make sense to have the bottleship and the bonsai tree use `START_TRACKING` /  `STOP_TRACKING` and then use that here as it'll be faster and also makes more sense.